src/main/java/com/kelinci/mainapp/api/RegisteredUserResponse.java

```diff
@@ -0,0 +1,13 @@
+package com.kelinci.mainapp.api;
+
+public class RegisteredUserResponse {
+    private String mail;
```

Comment:
IntelliJ coś tu podświetla? :) Jak tak, to proszę zmienić.

Reply:
done, zmienione na final